forms-html/rand_hewr/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+
+</head>
+
+<body>
+    <form>
+        <label for="fname">Enter your Email:</label><br>
+        <input type="text" id="fname" name="fname" required><br>
+        <label for="lname">Usser Name:</label><br>
+        <input type="text" id="lname" name="lname">
+        <br>
+        <label for="lname">Password:</label>
+        <br>
+        <input type="text" id="lname" name="lname"><br>
```

Comment:
You can't use the same id for more than one element on the page.
for more info: https://www.htmldog.com/guides/css/intermediate/classid/